Codegen/test/resources/output/TypeCobol/TypedefCopys/TypedefInsideCopy.rdz.tcbl

```diff
@@ -0,0 +1,20 @@
+﻿       identification division.
+       program-id. MyPGM.
+       environment division.
+       configuration section.
+       source-computer. ibm-370
+      *                 with debugging mode
+                        .
+       special-names. decimal-point is comma.
+       data division.
+       working-storage section.
+     
+
+       COPY MyCopyC. 
```

Comment:
As the COPY contains a Typedef, this line must be commented.

But you have chose a good test case.
Your COPY also contains data declaration outside the typedef.
So I'd say that this case must be refused. Codegen cannot comment inside the COY.


If we want to generated the code, then :
Typedef must be allowed in COPY if there are only typedef in the copy file.
Otherwise there must be an error (you cannot mix data declaration and typedef)


Note that for Cobol 2002 and if you don't want to generate code to Cobol 85, then you can mix data declaration and typedef in a COPY.

Reply:
We need to discuss this case at our next project meeting.